charts/k8ssandra/templates/stargate/service.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.stargate.enabled }}
+{{- if and (.Values.stargate.enabled) (.Values.cassandra.enabled) }}
```

Comment:
Do we really want to check if cassandra is enabled here?  Wouldn't this eliminate or certainly make harder the possibility to install the components separately and work with already installed components?

Reply:
Stargate requires our Cassandra to be up (it has a init container polling for pods that are up).

Reply:
That's true. I guess it would be additional work to be able to deploy Stargate separately on its own.